web/crux-ui/src/components/nodes/node-status-indicator.tsx

```diff
@@ -8,6 +8,8 @@ const statusToAssetName = (status: NodeStatus) => {
       return 'circle-red'
     case 'connected':
       return 'circle-green'
+    case 'outdated':
+      return 'circle-turquoise'
```

Comment:
Turquoise is used as a positive color on buttons. IMHO we should use purple as that is the obsolete deployment's color too.

Reply:
Implemented a `dyo-indicator` component instead of circle svgs, as discussed.